src/tools/components/AlignmentView.tsx

```diff
@@ -155,8 +189,8 @@ const MSAWrapper: React.FC<{
         </fieldset>
       </div>
       <div>
-        {activeView === View.overview ? (
-          <MSAView
+        {activeView === View.overview && tool === Tool.blast && (
```

Comment:
If Overview and Wrapped components take the same props, maybe it would be better to do, before the return:
```js
let Component;
if (activeView === View.overview) {
  Component = tool === Tool.blast ? BlastOverview : AlignOverview;
} else {
  Component = tool === Tool.blast ? BlastWrapped : AlignWrapped;
}
```
and in the return just use `<Component ...`

Reply:
Yes, I've incorporated your suggestion